chemprop/nn/transforms.py

```diff
@@ -0,0 +1,59 @@
+import torch
+from numpy.typing import ArrayLike
+from sklearn.preprocessing import StandardScaler
+from torch import Tensor, nn
+
+from chemprop.data.collate import BatchMolGraph
+
+
+class _ScaleTransformMixin(nn.Module):
+    def __init__(self, mean: ArrayLike, scale: ArrayLike, pad: int = 0):
+        super().__init__()
+
+        mean = torch.cat([torch.zeros(pad), torch.tensor(mean, dtype=torch.float)])
+        scale = torch.cat([torch.ones(pad), torch.tensor(scale, dtype=torch.float)])
+
+        if mean.shape != scale.shape:
+            raise ValueError(
+                f"uneven shapes for 'mean' and 'scale'! got: mean={mean.shape}, scale={scale.shape}"
+            )
+
+        self.register_buffer("mean", torch.tensor(mean, dtype=torch.float).unsqueeze(0))
+        self.register_buffer("scale", torch.tensor(scale, dtype=torch.float).unsqueeze(0))
```

Comment:
remnant of an old change. This can be changed now:
```suggestion
        self.register_buffer("mean", mean.unsqueeze(0))
        self.register_buffer("scale", scale.unsqueeze(0))
```